resources/l10n/uk_UA/LC_MESSAGES/loot.po

```diff
@@ -641,32 +641,36 @@ msgstr "Знайти попереднє"
 msgid "Find Next"
 msgstr "Знайти наступне"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:109
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:118
 msgid "Groups Editor"
 msgstr "Редактор груп"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:111
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:120
 msgid "Group name"
 msgstr "Ім'я групи"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:112
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:121
 msgid "Add a new group"
 msgstr "Додати нову групу"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:130
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:139
 msgid "No plugins are in this group."
 msgstr "У цій групі немає плагінів."
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:142
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:151
 #, c-format
 msgid "Plugins in %s"
 msgstr "Плагіни в %s"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:163
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:172
 msgid "Group already exists!"
 msgstr "Група уже існує!"
 
-#: src/gui/qt/plugin_editor/group_tab.cpp:92
+#: src/gui/qt/plugin_editor/group_tab.cpp:56
+msgid "%1 (Group does not exist)"
+msgstr "\"%1%\" (група не існує)"
```

Comment:
The `%1%` should be `%1`, the lack of a second `%` is annoying but necessary.

Reply:
Like this? (see new commit)

Reply:
Yep, thanks.